### PR TITLE
Handle session ✉️  (initial join + reconnect);

### PR DESCRIPTION
**Pull request by kirilknysh**

### CODE REVIEW

File: app/game/components/game-input/game-input.js [hunk 1]
```diff
@@ -10,10 +10,14 @@ import { verifySolution } from '../../action-creators/round';
 import './game-input.styl';
 
 class GameInput extends Component {
-    render({ result, dispatch, currentRoundIndex }) {
+    render({ result, dispatch, playerInput, currentRoundIndex }) {
         return (
             <div className="game-input">
-                <CodeEditor onChange={bindActionCreators(verifySolution, dispatch)} currentRoundIndex={currentRoundIndex} />
+                <CodeEditor
+                    onChange={bindActionCreators(verifySolution, dispatch)}
```
Comment (Shastel) [inline app/game/components/game-input/game-input.js]: is ir required to `bindActionCreator` here?

**Review by Shastel — CHANGES_REQUESTED** [review]

File: app/game/components/game-input/game-input.js (hunk 1 above)
Comment (kirilknysh) [reply app/game/components/game-input/game-input.js]: emm, yes. Otherwise the call of `dispatch` should be explicit.

Comment (lesnitsky) [reply app/game/components/game-input/game-input.js]: As per [redux docs](https://github.com/reactjs/react-redux/blob/master/docs/api.md#arguments) `connect` accepts `mapDispatchToProps` as second argument and will automatically bind dispatch to actions creators

Comment (kirilknysh) [reply app/game/components/game-input/game-input.js]: True, but there's also written that `bindActionCreators()` can be used to bind actions. Don't really matter for me. Fixed.

**Review by lesnitsky — APPROVED** [review]

**Review by Shastel — APPROVED** [review]

**Pull request merged**